### DISCUSSION TITLE
Highlight current day in datepicker

### DISCUSSION THREAD

**Discussion by laugri** [mantinedev/mantine#6435] [Feature requests]
First, thank you for your work, Mantine helps me daily !
In my opinion, the current date should be highlighted in the datepicker. It should be different from the selected date. I think it is a pretty common UI pattern that is expected by most people. I know I get comments about it not being there by users during usability tests !

Example from Material:

<img width="637" alt="image" src="https://github.com/mantinedev/mantine/assets/7806174/f10bd5e0-c855-4ccb-894b-85e7b27618b4">

**Comment by rtivital** [comment]
You can easily do that by applying styles to the `day` selector with `data-today` attribute.

**Reply by laugri** [reply]
Thanks for the answer. I understand it can be done manually. My point is that it should be the default behavior.

It is good UX to give users an immediate reference of "where they are" and it's a widely used pattern. Popular products such as  Google Calendar, Apple Calendar, Notion, Linear, have it. Shadcn, Tailwindcss, ant.design, and MUI implement it by default.

**Reply by rtivital** [reply]
The feature will be available in 7.11.0 minor release as `highlightToday` prop.

**Comment by andrejdukanic74** [comment]
@rtivital Hello
I need your help.
